pangeo_forge/recipe.py

```diff
@@ -146,10 +153,10 @@ class NetCDFtoZarrSequentialRecipe(BaseRecipe):
       `(ds: xr.Dataset, filename: str) -> ds: xr.Dataset`.
     :param process_chunk: Function to call on each concatenated chunk, with signature
       `(ds: xr.Dataset) -> ds: xr.Dataset`.
+    :param target_chunks: Desired chunk structure for the targret dataset.
     """
 
-    input_urls: Iterable[str] = field(repr=False)
-    sequence_dim: str
+    sequence_dim: str = ""
```

Comment:
What does an empty sting for sequence_dim mean? Does that mean the dimension name should literally be `""`? Or should we use an Optional[str] here?

Reply:
All parameters will need defaults if we want to use multiple inheritance / mixins without going crazy. I agree with Optional.